Frontend/src/App.jsx

```diff
@@ -1,31 +1,30 @@
-import { useState, createContext, useEffect } from "react";
+import { useState, useEffect } from "react";
 import Board from "./components/Board";
-import { boardDefault } from "./components/BoardDefault";
+import ErrorPopup from "./components/ErrorPopup";
 import GenerateWordSet from "./components/Words"
-import KeyBoard from "./components/KeyBoard";
 import GameOver from "./components/GameOver";
-import WordLength from "./components/WordLength";
-import ErrorPopup from "./components/ErrorPopup";
-
-
+import InputField from "./components/InputField";
+import guessWord from "./components/Feedback"
+import { boardDefault } from "./components/BoardDefault";
 
-export const AppContext = createContext();
 
 function App() {
-  const [board, setBoard] = useState(boardDefault);
-  const [startTime] = useState(new Date());
-  const [endTime, setEndTime] = useState(null)
-  const [currAttempt, setCurrAttempt] = useState({ attempt: 0, letterPos: 0 })
-  const [wordSet, setWordSet] = useState(new Set());
-  const [wrongLetters, setWrongLetter] = useState([]);
-  const [gameOver, setGameOver] = useState({ gameOver: false, guessedWord: false })
   const [correctWord, setCorrectWord] = useState("")
-  const [length, setLength] = useState(5);
+  const [wordSet, setWordSet] = useState(new Set())
```

Comment:
This is really "derived state", meaning that it's the result of taking the entire word array, and picking the subset of it that matches length. Avoid storing derived state as state, because it means that it's not theoretically possible, if you're not careful to always use `setLength` and `setWordSet` side by side, to end up in an application state where the two don't match, e.g. where `wordSet` contains a list of four-letter words but `length == 5`.